### DISCUSSION TITLE
KICAD_SYMBOL_DIR environment variable is missing

### DISCUSSION THREAD

**Discussion by Simple2024** [xesscorp/skidl#198] [Q&A]
Failed to configure the environment. The path has been added as required.
kicad v7.0.9     python3.12
WARNING: KICAD_SYMBOL_DIR environment variable is missing, so the default KiCad symbol libraries won't be searched.

**Comment by tapegoji** [comment]
need to set this up in Control Panel - users accounts -> Enviroments Vars. in windows
